### ISSUE TITLE
caja.desktop need include MimeType.

### ISSUE THREAD

**Issue by tumashu** [mate-desktop/caja#1619]
like:

MimeType=inode/directory;

**Comment by tumashu** [comment]
It is used by  mate-default-applications

**Comment by rbuj** [comment]
The MimeType key has another purpose, you can find more information at https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html

**Comment by rbuj** [comment]
Anyway, this key has already been defined:
```
$ xdg-mime query default inode/directory
caja-folder-handler.desktop
```
https://github.com/mate-desktop/caja/blob/7d64342c6914b8f502512fd111ec3acb2083741e/data/caja-folder-handler.desktop.in.in#L12

**Comment by tumashu** [comment]
Yes, when I restart my computer, I find that caja show in mate-default-applications.

**Comment by tumashu** [comment]
![图片](https://user-images.githubusercontent.com/67725/155877512-67c4dc28-0b4e-4d9a-992e-1d0e0cc07802.png)
